frontend/packages/ceph-storage-plugin/src/components/odf-system/utils.ts

```diff
@@ -0,0 +1,49 @@
+import { HumanizeResult } from '@console/internal/components/utils';
+import { PrometheusResponse } from '@console/internal/module/k8s';
+
+// Operator uses`<kind>.<apiGroup>/<apiVersion>` don't ask why
+export const getGVK = (weirdLabel: string) => {
+  const kind = weirdLabel.slice(0, weirdLabel.indexOf('.'));
+  const apiGroup = weirdLabel.slice(weirdLabel.indexOf('.') + 1, weirdLabel.indexOf('/'));
+  const apiVersion = weirdLabel.slice(weirdLabel.indexOf('/') + 1, weirdLabel.length);
+  return { kind, apiGroup, apiVersion };
+};
+
+export type SystemMetrics = {
+  metrics: {
+    [systeName: string]: {
+      rawCapacity: HumanizeResult;
+      usedCapacity: HumanizeResult;
+      iops: HumanizeResult;
+      throughput: HumanizeResult;
+      latency: HumanizeResult;
+    };
+  };
+};
+
+type MetricNormalize = (
+  latency: PrometheusResponse,
+  throughput: PrometheusResponse,
+  rawCapacity: PrometheusResponse,
+  usedCapacity: PrometheusResponse,
+  iops: PrometheusResponse,
+) => SystemMetrics['metrics'];
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const normalizeMetrics: MetricNormalize = (
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  _latency,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  _throughput,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  _rawCapacity,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  _usedCapacity,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  _iops,
+) => {
+  // Todo(bipuladh): Add parsing logic for above items
+  return {
+    metrics: {} as any,
+  };
+};
```

Comment:
add when ready

Reply:
Let's keep it as it is now. It's a skeleton and rest of parsing is dependent on this function.